screen/transactions/src/main/java/com/ivy/transactions/TransactionsViewModel.kt

```diff
@@ -89,7 +89,7 @@ class TransactionsViewModel @Inject constructor(
     private val calcAccIncomeExpenseAct: CalcAccIncomeExpenseAct,
     private val calcTrnsIncomeExpenseAct: LegacyCalcTrnsIncomeExpenseAct,
     private val exchangeAct: ExchangeAct,
-    private val transactionWriter: WriteTransactionDao,
+    private val transactionWriter: TransactionRepositoryImpl,
```

Comment:
Never use the implementation directly. You should use `TransactionRepository` which is the abstraction

Reply:
Damn!😅
Sorry i overlooked it by mistake, i will revert it.